scan-external-contacts/config.yml

```diff
@@ -0,0 +1,146 @@
+---
+title: Scan through External Contacts
+description: See how to scan through external contacts, external organizations, relationships and notes using the External Contacts API.
+categories:
+- Contacts
+steps:
+- title: Introduction
+  content: |
+    This tutorial walks through the steps of implementing a function to scan through the external contacts of a Genesys Cloud organization, and to process them before saving them into a file.
+
+    There are 4 types of objects that can be retrieved using the External Contacts Scanning API: External Contacts, External Organizations, Relationships, and Notes.
+
+    Go here for more information about [External Contacts Scanning](/api/rest/v2/externalcontacts/scan.html).
+
+    For this tutorial, we are going to use the Genesys Cloud SDK, and we will scan and export the External Contacts.
+    
+    The sample code is using javascript async/await (making it synchronous) to ease readibility. You can change and update it based on your needs.
+
+- title: Authentication
+  content: |
+    You will need an OAuth client using Client Credentials to authenticate. See [Create an OAuth Client](https://help.mypurecloud.com/articles/create-an-oauth-client/) for more info
```

Comment:
This makes it sound like these APIs can only be used with client credentials. Assuming that's not the case, can you reword to be more generic so the statement works for any client type? Or be more explicit that the tutorial is using client credentials, but you can use user-based auth as well?

Reply:
One thing you might want to note too is that too many people use client credential grants because they are the easiest to understand.  The reality is that if you are going to be calling this as a service from a web facing application you should be using a client auth.